structuredevent-model/src/main/java/com/sequenceiq/environment/parameter/dto/AzureResourceEncryptionParametersDto.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.environment.parameter.dto;
+
+public class AzureResourceEncryptionParametersDto {
+    private final String keyUrl;
```

Comment:
If we decide in favor of using Vault for this, I wonder if it should already happen in the DTO or only later in the entity. Should be checked what the convention is.

Reply:
Working on this part, will update the PR.

Reply:
Okay, so let us do the `Secret` conversion only at the entity level.